blackduck/HubRestApi.py

```diff
@@ -1223,6 +1239,16 @@ def get_ldap_configs(self):
         response = requests.get(url, headers=headers, verify = not self.config['insecure'])
         jsondata = response.json()
         return jsondata
+
+    def delete_unmapped_codelocations(self, limit=1000):
+        jsondata = self.get_codelocations(limit, True)
+        codelocations = jsondata['items']
+        for c in codelocations:
+            if 'mappedProjectVersion' not in c:
```

Comment:
Is this redundant? 
code locations returned by get_codelocations() with unmapped=True should already return only those code locations that are empty